docs/03-extensions/magento1/reference/axios-instances.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: "Axios instances"
+sidebar_position: 1
+description: "This reference explains how to create `axios` instances to interact with Magento's API."
+---
+
+<p>{frontMatter.description}</p>
+
+
+## `makeAdminClientFromRequest`
+
+The `makeAdminClientFromRequest` function creates an `axios` instance that can be
+used to interact with Magento's API as an Admin.
+
+```js
+makeAdminClientFromRequest(request);
+```
+
+Arguments:
+
+| Name      | Type    | Description                  |
+| :-------- | :------ | :--------------------------- |
+| `request` | Request | v2 compatible request object |
+
+Example:
+
+```js
+import { makeAdminClientFromRequest } from "@front-commerce/magento1/axios";
+
+export default {
+  namespace: "Acme/Feature",
+  contextEnhancer: ({ req }) => {
+    const adminAxiosInstance = makeAdminClientFromRequest(req);
+
+    // rest of logic
+  },
+};
+```
+
+## `makeAuthServiceFromRequest`
```

Comment:
**issue(debatable?):** `authService` has nothing to do with axios, although it was migrated in `magento(1|2)/axios`. Is it really the right place for it?

The documentation part is good otherwise.

Reply:
Since it has been exposed publicly from axios export, i think it should be documented here. But i approve that a such export doesn't belong to axios package